compiler/rustc_hir_analysis/src/lib.rs

```diff
@@ -213,6 +213,12 @@ pub fn check_crate(tcx: TyCtxt<'_>) -> Result<(), ErrorGuaranteed> {
         tcx.hir().for_each_module(|module| tcx.ensure().check_mod_item_types(module))
     });
 
+    if tcx.features().rustc_attrs {
+        tcx.sess.track_errors(|| {
+            tcx.sess.time("variance_testing", || collect::test_opaque_hidden_types(tcx));
```

Comment:
wrong name in `sess.time`, probably easier to simply not time it?

Reply:
true